optimum/exporters/openvino/model_patcher.py

```diff
@@ -2712,7 +2712,14 @@ def patched_forward(*args, **kwargs):
 
```

Comment:
Looks like the mismatch for qwen2_vl is coming from https://github.com/huggingface/transformers/pull/33487/files (mismatch between the `position_ids` when not given at inference)